org.eclipse.transformer/src/main/java/org/eclipse/transformer/action/ElementAction.java

```diff
@@ -13,7 +13,12 @@
 
 import org.eclipse.transformer.TransformException;
 
+import java.util.regex.Pattern;
+
 public interface ElementAction extends Action {
+
+	Pattern SIGNATURE_FILE_PATTERN = Pattern.compile("META-INF/[A-Za-z0-9_-]+\\.(SF|RSA|DSA|EC)");
```

Comment:
Also `META-INF/SIG-*`. See https://docs.oracle.com/en/java/javase/21/docs/specs/jar/jar.html#signed-jar-file

Reply:
Borrowed the pattern string from `aQute.bnd.osgi.Jar.METAINF_SIGNING_P`